dao/application_authentication_dao_test.go

```diff
@@ -243,3 +243,38 @@ func TestApplicationAuthenticationsByAuthenticationsDatabase(t *testing.T) {
 
 	DropSchema("appauthfind")
 }
+
+// TestApplicationAuthenticationListOffsetAndLimit tests that List() in app auth dao returns correct count value
+// and correct count of returned objects
+func TestApplicationAuthenticationListOffsetAndLimit(t *testing.T) {
```

Comment:
For future reference - I think these should have been DAO tests since this functionality is handled by the DAO and not the handlers (e.g. we use the List functions from other parts of the code like the availability status listener or the job engine)

Merging anyway to fix the UI.